apps/sv/frontend/src/__tests__/sv.test.tsx

```diff
@@ -50,6 +54,64 @@ describe('SV user can', () => {
     await screen.findByText('You are on ScratchNet');
   });
 
+  test('browse to the validator onboarding tab', async () => {
+    const user = userEvent.setup();
+    render(<AppWithConfig />);
+
+    expect(await screen.findByText('Validator Onboarding')).toBeDefined();
+    await user.click(screen.getByText('Validator Onboarding'));
+
+    expect(await screen.findByText('Validator Onboarding Secrets')).toBeDefined();
+  });
+
+  test('create a new validator secret with party hint', async () => {
+    const user = userEvent.setup();
+    render(<AppWithConfig />);
+
+    expect(await screen.findByText('Validator Onboarding')).toBeDefined();
+    await user.click(screen.getByText('Validator Onboarding'));
+
+    const partyHintInput = screen.getByTestId('create-party-hint');
+    await user.type(partyHintInput, 'wrong-input');
+
+    expect(screen.getByTestId('create-validator-onboarding-secret').hasAttribute('disabled')).toBe(
```

Comment:
Surprised this works for you. I've had issues with hasAttribute so I stayed away from it.